pkg/_popularity/lib/popularity.dart

```diff
@@ -70,10 +70,9 @@ class VoteData {
 
   int get score => _score(this);
 
-  VoteData(this.direct, this.dev, this.total) {
-    assert(direct >= 0 && direct <= total);
-    assert(dev >= 0 && dev <= total);
-  }
+  VoteData(this.direct, this.dev, this.total)
+      : assert(direct >= 0 && direct <= total),
```

Comment:
Same here. My reasoning: while asserts here are simple, we may put more complex checks in them, and initializing block may not be the best place for it.